luwak/src/main/java/uk/co/flax/luwak/termextractor/treebuilder/TreeBuilders.java

```diff
@@ -55,32 +54,32 @@ public QueryTree buildTree(QueryAnalyzer builder, Query query) {
     public static final List<QueryTreeBuilder<? extends Query>> DEFAULT_BUILDERS = CollectionUtils.makeUnmodifiableList(
             new BooleanQueryTreeBuilder(),
             newConjunctionBuilder(PhraseQuery.class,
-                    (b, q) -> Arrays.asList(q.getTerms()).stream().map(TermNode::new).collect(Collectors.toList())),
+                    (b, w, q) -> Arrays.asList(q.getTerms()).stream().map(qq -> new TermNode(qq, w)).collect(Collectors.toList())),
```

Comment:
Instead of Arrays.asList(q.getTerms()).stream(), Arrays.stream(q.getTerms()) can be used - this happens on a few lines in this class.